spec/requests/admin/api_controller_spec.rb

```diff
@@ -82,7 +134,44 @@
       end
     end
 
-    describe "#destroy" do
+    shared_examples "key update impossible" do
```

Comment:
```suggestion
    shared_examples "key update not allowed" do
```

`impossible` feels a little too strong IMO. What do you think about the suggestion above?

Reply:
Sure. That works too. Yes, I agree it's quite strong, but I think for an access related spec being explicit about the possibility does convey the message too.